airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -472,7 +472,7 @@ def _change_state(
         if self.kube_config.delete_worker_pods:
             if state != TaskInstanceState.FAILED or self.kube_config.delete_worker_pods_on_failure:
                 self.kube_scheduler.delete_pod(pod_name=pod_name, namespace=namespace)
-                self.log.info("Deleted pod: %s in namespace %s", key, namespace)
+                self.log.info("Deleted pod associated with the TI %s. Pod name: %s. Namespace: %s", key, pod_name, namespace)
```

Comment:
using f-string might be good idea here. It is more readable.
```suggestion
                self.log.info(f"Deleted pod associated with the TI {key}. Pod name: {pod_name}. Namespace: {namespace}")
```

Reply:
Since we enabled `G` of the `ruff` rules, we should not use f-string in the logs.
see more: 
1. https://docs.astral.sh/ruff/rules/logging-f-string/
2. https://github.com/apache/airflow/blob/a7b0dc25c0201e3e69726aa6033102128d8d9046/pyproject.toml#L267